administration/src/components/applications/ApplicationUserOverviewController.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import { NonIdealState, Spinner } from '@blueprintjs/core'
+
+import { useGetApplicationByUserAccessKeyQuery } from '../../generated/graphql'
+import ErrorHandler from '../../ErrorHandler'
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import { ApplicationViewComponent } from './ApplicationsOverview'
+import { useAppToaster } from '../AppToaster'
+
+const NotFound = styled(NonIdealState)`
+  margin: auto;
+`
+
+const CenteredMessage = styled(NonIdealState)`
+  margin: auto;
+`
+
+const ApplicationUserOverviewController = (props: { providedKey: string }) => {
+  const [withdrawed, setWithdrawed] = useState<boolean>(false)
+  const appToaster = useAppToaster()
+  const { loading, error, data, refetch } = useGetApplicationByUserAccessKeyQuery({
+    variables: { accessKey: props.providedKey },
+    onError: error => {
+      console.error(error)
+      appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+    },
+  })
+
+  if (loading) return <Spinner />
+  else if (error || !data) return <ErrorHandler refetch={refetch} />
+  if (data.application.withdrawalDate) return <CenteredMessage title='Ihr Antrag wurde bereits zurückgezogen' />
+  if (withdrawed) return <CenteredMessage title='Ihr Antrag wurde zurückgezogen' />
+  else {
+    return (
+      <ApplicationViewComponent
+        application={data.application}
+        gotConfirmed={() => setWithdrawed(true)}
+        actionType={'withdraw'}
+      />
```

Comment:
Some feedback:
* Maybe some context is missing like a title
`Ihr Antrag auf die Bayerische Ehrenamtskarte`

*  Also, the Antrag should already be expanded (and probably there should not be an option to shrink it).

* We should add some text to the withdrawing button for clarification. Like:

{SECTION_DIVIDER}
Hier können Sie Ihren Antrag zurückziehen und Ihre Eingaben unwiderruflich löschen.

{Button: Antrag Zurückziehen}


* Also, I feel like we should compose these components here instead of pushing all this functionality into the ApplicationViewComponent.

* The site should work on small devices (currently the width is not responsive).

Reply:
alright i thought its a good idea to reuse sth quite similar and we can reuse for multiple actions but i can also create a separate component which i had in the beginning